src/services/expenses.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const { getNewId } = require('../getNewId');
+
+let expenses = [];
+
+const getInitial = () => {
+  expenses = [];
+
+  return expenses;
+};
+
+const getAllExpenses = ({
+  userId,
+  categories,
+  from,
+  to,
+}) => {
+  let visibleExpenses = expenses;
+
+  if (userId) {
+    visibleExpenses = expenses.filter(expense => expense.userId === +userId);
+  }
+
+  if (categories) {
+    visibleExpenses = expenses.filter(
+      ({ category }) => categories.includes(category)
+    );
+  }
+
+  if (from) {
+    visibleExpenses = expenses.filter(({ spentAt }) => spentAt > from);
+  }
```

Comment:
Why can't I combine filter options? Let's instead of filtering initial array filter already filtered one